pytest.ini

```diff
@@ -3,6 +3,7 @@
 markers =
     functional: functional tests
     database: database tests
+# need pytest-cov to do --cov=arkane,rmgpy (?)
```

Comment:
What does this mean? What is it for?

Reply:
that was a note I left for myself and committed accidentally. a reminder that we need to have `pytest-cov` installed to run the tests. I later found that you could have a required plugins part of the config file so now it lives there